dist/images/kubectl-ko

```diff
@@ -263,6 +263,14 @@ checkDeployment(){
   fi
 }
 
+checkKubeProxy(){
+  healthResult=`kubectl get node -o wide | grep -v "INTERNAL-IP" | awk '{printf "curl -sL -w %{http_code} http://%s:10256/healthz -o /dev/null | grep -v 200 \n", $6}' | bash` || true
```

Comment:
ipv6 e2e test failed. Should use `curl -g -6` for ipv6 cluster

Reply:
@oilbeater Thanks for your reply! However, I don’t have an ipv6 cluster. What I want to know is whether `curl -g -6` is also valid for ipv4 cluster. Should I need different commands for these two clusters?

Reply:
After some tests, these options also work for ipv4 cluster. We can go on